src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/__tests__/autocomplete.command.stats.test.ts

```diff
@@ -555,7 +555,7 @@ describe('autocomplete.suggest', () => {
         });
 
         test('suggests `?interval` option', async () => {
-          const suggestions = await suggest('FROM a | STATS BY BUCKET(@timestamp, /)', {
+          const suggestions = await suggest('FROM index_a | STATS BY BUCKET(@timestamp, /)', {
```

Comment:
why this and other changes to the index patterns in tests?

Reply:
I explain it in the comments and in the description!